index.html

```diff
@@ -33,9 +33,9 @@ <h4 class="service-heading">Cloud Architecture</h4>
           <i class="fa fa-circle fa-stack-2x text-primary"></i>
           <i class="fa fa-code fa-stack-1x fa-inverse"></i>
         </span>
-        <h4 class="service-heading">Puppet Module Development</h4>
+        <h4 class="service-heading">Puppet Consulting</h4>
         <p class="text-muted">
-          We specialize in building Puppet modules to meet your specific needs.
+          We specialize in <a href="./puppet_consulting/">building Puppet modules to meet your specific needs.</a>
```

Comment:
TODO: Revise this section to link up with the new consulting page. This link was added to verify the ability to link to the new Puppet consulting page, and should not be kept without revising the section.